manager/manager/templatetags/manager.py

```diff
@@ -232,3 +232,13 @@ def has_expired(errors) -> bool:
 
 
 register.filter("has_expired", has_expired)
+
+@register.simple_tag
+def url_replace(request, field, value):
```

Comment:
This is gonna be used only for sorting ; maybe we can drop the `field` param and use this function only to update sorting